config/schema_helpers.go

```diff
@@ -60,26 +60,49 @@ func setupSchemaLoaders(schemaData string, schema *map[string]interface{}, schem
 	return nil
 }
 
+func oneOfValidTypes(oneOf interface{}, validTypes []string) []string {
```

Comment:
This function name is not indicative of what it does. How about `appendValidType` or something like that? I want to communicate that it is appending the the slice passed in